### PR TITLE
11-13 release notes 1

### PR DESCRIPTION
**Pull request by kaarthis**
Please review and approve.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. 
+* AKS begins pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure NAT gateways do not support IPv6 and therefore cannot be used with dual-stack clusters as the cluster would not have a valid IPv6 outbound connection.
+* [AKS clusters with Calico enabled](https://github.com/Azure/AKS/issues/3315) should not upgrade to Kubernetes v1.25 preview.
+* Starting Kubernetes v1.25 two in-tree driver persistent volumes wont be supported in AKS : kubernetes.io/azure-disk, kubernetes.io/azure-file.
```
Comment (shashankbarsin) [inline CHANGELOG.md:15]: do we have link to any migration guidance? @olsenme

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. 
+* AKS begins pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure NAT gateways do not support IPv6 and therefore cannot be used with dual-stack clusters as the cluster would not have a valid IPv6 outbound connection.
+* [AKS clusters with Calico enabled](https://github.com/Azure/AKS/issues/3315) should not upgrade to Kubernetes v1.25 preview.
```
Comment (shashankbarsin) [inline CHANGELOG.md:14]: I believe this is being addressed in 11/27 release. we need to publish notes for 11/27 too. Can we check on the release status of this with engineering and update the notes?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-11-13
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is retiring `v1.22.x` on December 4th 2022. Please [upgrade](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cl) your clusters to `v1.23` and above.
```
Comment (shashankbarsin) [inline CHANGELOG.md:9]:
```suggestion
* AKS is retiring `v1.22.x` on December 4th 2022. Please [upgrade](https://learn.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) your clusters to `v1.23` and above.
```

File: CHANGELOG.md (hunk 1 above)
Comment (kaarthis) [reply CHANGELOG.md:15]: Nothing yet hopefully by end of the week is what @olsenme  mentioned.

**Pull request closed without merging**